### PR TITLE
Adding --vcfinput flag to pathogenicity

### PR DESCRIPTION
**Pull request by sakshamphul**
[For this ticket](https://app.zenhub.com/workspaces/d3b-bfx-5cdd720eed0dce4209e0b8a5/issues/gh/d3b-center/bixu-tracker/1818) : This PR adds --vcfinput flag to annotate vcf and output it.  

Test runs:

1) https://cavatica.sbgenomics.com/u/d3b-bixu/d3b-diskin-pathogenicity-dev/tasks/1d071b30-3d43-4eac-af85-c0b2b8e0a0c8/
2) https://cavatica.sbgenomics.com/u/d3b-bixu/d3b-diskin-pathogenicity-dev/tasks/2ab3adf9-3f94-4fdb-a32e-2a687e9a9f83/
3) https://cavatica.sbgenomics.com/u/d3b-bixu/d3b-diskin-pathogenicity-dev/tasks/641329d4-2104-4c2f-89c7-6b0a83132c08/

### CODE REVIEW

**Review by dmiller15 — APPROVED** [review]

**Review by migbro — APPROVED** [review]
This seems to do what we want, however, I really think @naqvia should take a look first before merging to check that the outputs from the wf fix his issue

**Comment by naqvia** [comment]
Looks good to me! Thank you.

**Pull request merged**
